### PR TITLE
Lint codebase

### PR DESCRIPTION
**Pull request by HaltonO**
Only blank spaces & semicolons removed in the process.

### CODE REVIEW

File: imports/ui/SpaceChat/index.js [hunk 1]
```diff
@@ -1,41 +1,31 @@
...
-    );
+    )
   }
 }
-
-
-
-
-/*<div>
+/*
```
Comment (madeline-petersen) [inline imports/ui/SpaceChat/index.js]: Can you remove this whole commented-out block while you're at it? :)

Comment (HaltonO) [reply imports/ui/SpaceChat/index.js]: yes of course :)

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
```
Comment (madeline-petersen) [inline .eslintrc.js]: We can delete this file and use .eslintrc.json instead

**Comment by madeline-petersen** [comment]
Let's use the original package.json file from the meteor-eslint branch.  To pull this in, do

```
git checkout lint-codebase
git checkout meteor-eslint — package.json
```

**Comment by madeline-petersen** [comment]
One of the warnings is we're getting on a lot of our components is

```
2:16  warning  Component should be written as a pure function     react/prefer-stateless-function
```

Here's how I fixed it for the Splash component.  Basically, I replaced 
`export default class Splash extends Component {`
with
`export default function Splash() {`
and removed the render function.

Like this...

```
import React from 'react';

export default function Splash() {
  return (
    <div>
      ...
    </div>
  );
}
```

**Pull request closed without merging**